osv/models.py

```diff
@@ -337,7 +337,7 @@ def id(self):
 
   @property
   def repo_url(self):
-    """Repo URL."""
+    """Repo URL. Assumes there is only ever one repo URL per advisory."""
```

Comment:
It's probably extremely unlikely an advisory would have more than one git repo, but I wonder if we should think about storing the repo in the AffectedCommits, and do the filtering logic when querying those.